### PR TITLE
UX-550# Pipeline API: /pipelines

### PR DESCRIPTION
**Pull request by vivek**
Related to issue # . 

Summary of this pull request: 
.
.
.

@reviewbybees

### CODE REVIEW

**Comment by imeredith** [comment]
@vivek @i386 Why are we doing this? In a recent discussion about queued items it was decided not to global scope them... so why pipelines?

File: blueocean-rest/README.md [hunk 1]
```diff
@@ -107,6 +109,23 @@ $$
       } 
     ]
 
+## Get Pipelines
+
+Pipelines are sorted by pipeline name alphabetically
+
+    curl -v -X GET  http://localhost:8080/pipelines/
```
Comment (imeredith) [inline blueocean-rest/README.md]: should be /rest/pipelines/ ?

**Comment by michaelneale** [comment]
@imeredith the feature request was specifically for pipelines across organisations. It isn't the main pipeline api that is currently in use.

**Comment by imeredith** [comment]
@michaelneale i realise that, i just wonder why we want one and not the other.

File: blueocean-rest/README.md (hunk 1 above)
Comment (vivek) [inline blueocean-rest/README.md]: yes, good catch, will update it.

**Comment by vivek** [comment]
@imeredith see https://cloudbees.atlassian.net/browse/UX-310, thats driving this API, that is list of pipelines across organizations.

**Comment by michaelneale** [comment]
@imeredith I believe we want both

**Comment by imeredith** [comment]
@vivek I know, it just doesn't make sense to me as we seemed to be scoping everything to org, including queued items.

**Comment by imeredith** [comment]
@michaelneale probably, it just struck me as odd after last weeks discussions

**Comment by imeredith** [comment]
🐝 from me, as long as @i386 is happy with the global scope, which he should be since i assumed he was one that specc'ed it :)

**Comment by i386** [comment]
@imeredith yes, happy with this. @kohsuke has known this is coming for a while :)

**Pull request merged**